src/main/java/nextstep/ladder/domain/Line.java

```diff
@@ -0,0 +1,17 @@
+package nextstep.ladder.domain;
+
+public enum Line {
+    INITLINE(false),
+    NONELINE(false),
+    ISLINE(true);
```

Comment:
```suggestion
    IN_IT_LINE(false),
    NONE_LINE(false),
    IS_LINE(true);
```
이렇게 구분해줘야 읽기 쉬울 것 같아요!